.travis.yml

```diff
@@ -41,6 +41,7 @@ before_install:
 
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
+  - php bin/update_hostname_validator.php --check-only
```

Comment:
Another thing, I preferred to run the check right after install because I thought that one could prefer to see the build fail earlier as possible but this may vary on what one is looking for.